lib/rspec/core/output_wrapper.rb

```diff
@@ -0,0 +1,26 @@
+module RSpec
+  module Core
+    # Wrapper around an output stream object.
+    class OutputWrapper
+      # @private
+      attr_accessor :output
```

Comment:
Given you use the instance variable directly and don't use the reader, should this be just `attr_writer`?